src/LiteCore/src/LiteCore.Shared/Interop/Fleece_native.cs

```diff
@@ -198,6 +277,50 @@ public static string FLDictKey_GetString(FLDictKey* dictKey)
         [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
         public static extern FLValue* FLDict_GetWithKey(FLDict* dict, FLDictKey* dictKey);
 
+        [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
+        public static extern FLMutableDict* FLDict_MutableCopy(FLDict* source, FLCopyFlags x);
+
+        [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
+        public static extern FLMutableDict* FLMutableDict_New();
+
+        [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
+        public static extern FLDict* FLMutableDict_GetSource(FLMutableDict* x);
+
+        [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
+        [return: MarshalAs(UnmanagedType.U1)]
+        public static extern bool FLMutableDict_IsChanged(FLMutableDict* x);
+
+        public static FLSlot* FLMutableDict_Set(FLMutableDict* FL_NONNULL, string key)
```

Comment:
This is really funny...I'll need to fix this script later to ignore the `FL_NONNULL` macro